Jenkinsfile

```diff
@@ -45,7 +45,7 @@ try {
           rosEnv = docker.build 'ros:snapshot', "--build-arg ROS_DE_VERSION=${rosDeVersion} tools/sync_test_server"
         }
 
-	    rosContainer = rosEnv.run('-v /tmp=/tmp/.ros')
+	    rosContainer = rosEnv.run()
```

Comment:
If you mount to something in the workarea, you would be able to get the ros log normally instead of with `docker cp`, which will only run if the ros container is still running.

Reply:
I'm guessing that was the reason we did it in the first place, but as you pointed out it leaves the possibility for something else to change the state of ROS by accident.

Judging from our integration tests, it does appear that something is happening if a shared `tmp` folder is used between ROS instances, I'm not entirely sure what though.

Also given that we manually close the ros container https://github.com/realm/realm-java/blob/master/Jenkinsfile#L120 we should be reasonable sure to get the logs for most cases we care about.

Reply:
You don't need to mount something globally available like /tmp. You could mount `$(pwd)/roslogs` or whatever in your own private workarea